README.md

```diff
@@ -25,15 +25,15 @@ up MEDS, we will define some key terminology that we use in this standard.
 ## Core MEDS Data Organization
 
 MEDS consists of four main data components/schemas:
-  1. A _data schema_. This schema describes the underlying medical data, organized as sequences of patient
+  1. A _data schema_. This schema describes the underlying medical data, organized as sequences of subject
      observations, in the dataset.
-  2. A _patient subsequence label schema_. This schema describes labels that may be predicted about a patient
-     at a given time in the patient record.
+  2. A _subject subsequence label schema_. This schema describes labels that may be predicted about a subject
+     at a given time in the subject record.
   3. A _code metadata schema_. This schema contains metadata describing the codes used to categorize the
```

Comment:
We should add a note here about how all codes should be in this table.

Reply:
Done.